internal/router/middlewares/contextkeys.go

```diff
@@ -4,10 +4,10 @@ package middlewares
 type ContextKey int
 
 const (
-	// ContextKeyAddress is used to store the address of the client for the incoming request.
```

Comment:
The term "address" in this comment was confusing to me at first. I believe it is the wallet address, not the IP address, since the IP address is found in ContextIPAddress.
As such we can remove it because the siwe was the only place the wallet address was available in the request context.